spec/interactors/work/refresh_metadata_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe Work::RefreshMetadata do
+  let(:owner) { User.find_by(login: OWNER) }
+  let(:collection) { create(:collection, owner_user_id: owner.id) }
+  let(:original_metadata) { [{ label: 'refresh-metadata-test', value: 'RFSH-M3DA' }].to_json }
+  let(:at_id) { 'http://example.com/manifest' }
+  let(:v3_hash) do
+    {
+      id: at_id,
+      label: {
+        values: ['refresh-metadata-test']
+      },
+      metadata: original_metadata
+    }.to_json.to_s
```

Comment:
Is `to_s` redundant?

Reply:
In this case no. The params being passed needs to be in .to_json and then to_s, as it will be parsed again later on JSON.parse(original_metadata) inside the sc_manifest.

That being said I can write the params in pure strings, but for the sake of readability here I decided to do this.

Reply:
I like this.  (I was just curious about the `to_s`.)